modular_zubbers/modules/vending/vending.dm

```diff
@@ -0,0 +1,45 @@
+//Taken from Modular Vending in Skyrat
+
+/obj/machinery/vending
+	/// Additions to the `products` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
+	var/list/zubbers_products
+	/// Additions to the `product_categories` list of the vending machine, modularly. Will become null after Initialize, to free up memory.
+	var/list/zubbers_product_categories
+	/// Additions to the `premium` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
+	var/list/zubbers_premium
+	/// Additions to the `contraband` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
+	var/list/zubbers_contraband
```

Comment:
I could've sworn we already had this somewhere. Guess not. alas, add a whitespace
```suggestion
/obj/machinery/vending
	/// Additions to the `products` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
	var/list/zubbers_products
	/// Additions to the `product_categories` list of the vending machine, modularly. Will become null after Initialize, to free up memory.
	var/list/zubbers_product_categories
	/// Additions to the `premium` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
	var/list/zubbers_premium
	/// Additions to the `contraband` list  of the vending machine, modularly. Will become null after Initialize, to free up memory.
	var/list/zubbers_contraband

```